projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -146,23 +195,32 @@ export class IgxCarouselComponent implements OnDestroy {
     private _destroyed: boolean;
     private _total = 0;
 
-    constructor(private element: ElementRef) {}
-
+    constructor(private element: ElementRef) { }
+    /**
+     *@hidden
+     */
     public ngOnDestroy() {
         this._destroyed = true;
         if (this._lastInterval) {
             clearInterval(this._lastInterval);
         }
     }
-
+    /**
+     * Returns a string containing the position of the passed slide and the total number of slides.
+     * ```typescript
+     * let slidePosition = this.carousel.setAriaLabel(this.slide)
+     * ```
+     * @memberof IgxCarouselComponent
+     */
     public setAriaLabel(slide) {
```

Comment:
This one should be hidden. When you notice such public methods that do not make sense to be available ( why would someone pass the slide to receive a string like "1 of 5" ), consider if they are public only for the sake of being available in the templates. Checking the carousel templates proves this - <li *ngFor="let slide of slides" [attr.aria-label]="setAriaLabel(slide)"